app/lib/ds-routing/config/yesNoPathsConfigModel.js

```diff
@@ -25,5 +25,5 @@ var yesNoPathsConfigSchema = new mongoose.Schema({
 })
 
 module.exports = function(connection) {
-  return connection.model('yes_no_paths', yesNoPathsConfigSchema);
+  return connection.model('yes_no_path', yesNoPathsConfigSchema);
```

Comment:
was there a reason why you didn't change the string to `app.ConfigName.YES_NO_PATHS` here?